### PR TITLE
Add our CoC diffs

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @metajack

This just clarifies that we are the point of contact for Servo CoC issues, not the Rust Moderation team :-)

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9744)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ For specific git instructions, see [GitHub workflow 101](https://github.com/serv
 
 ## Conduct
 
-We follow the [Rust Code of Conduct](http://www.rust-lang.org/conduct.html).
+In all Servo-related forums, we follow the [Rust Code of Conduct](http://www.rust-lang.org/conduct.html). For escalation or moderation issues, please contact Jack (jmoffitt@mozilla.com) and Lars (larsberg@mozilla.com) instead of the Rust moderation team.
```
Comment (KiChjang) [inline CONTRIBUTING.md]: Is his email jack@mozilla.com instead?

Comment (metajack) [inline CONTRIBUTING.md]: Both work :)

**Comment by Ms2ger** [comment]
@bors-servo retry
* #9033

**Pull request merged**